tests/httpserver.py

```diff
@@ -7,6 +7,18 @@
 from six.moves import socketserver
 
 
+class TestsHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
```

Comment:
Why is it necessary to replace SimpleHTTPServer?

Reply:
@danrodrig is not replacing anything, he is extending functionality for his testing .......

Reply:
@cbouchar 
> Why is it necessary to replace SimpleHTTPServer?

I don't understand the question. Where is SimpleHTTPServer replaced?

I'm defining the TestsHandler class here. It inherits from SimpleHTTPRequestHandler class, which is defined in the SimpleHTTPServer module.

Reply:
Argh!  I'm definitely rustly on my python.

Reply:
So Carol are you saying it is time for rusty lang https://www.rust-lang.org/ 👀